### ISSUE TITLE
IE 11 - no example seems to work

### ISSUE THREAD

**Issue by faceplant23** [pa7/heatmap.js#172]
Dear all,
Tested it on two machines with IE 11 installed: the heatmap appears on only some provided examples at http://www.patrick-wied.at/static/heatmapjs/examples.html - Unfortunately without any JS error, so I have no chance to analyze further. For me the Maps examples are the most interesting, which also do not work anymore in IE 11. On all other browsers and on IE 9 it still works fine.
Thanks for your help.

**Comment by pa7** [comment]
Could you tell me which examples worked for you?

**Comment by faceplant23** [comment]
for sure: http://www.patrick-wied.at/static/heatmapjs/example-click-heatmap.html , http://www.patrick-wied.at/static/heatmapjs/example-mousemove-heatmap.html work - the rest unfortunately not in IE11

**Comment by faceplant23** [comment]
Hey Patrick,
were you able to find out something or at least were you able to reproduce the problem?
Is there any chance to help? Maybe by donation or something?
Thanks

**Comment by pa7** [comment]
sorry I haven't had a chance to test it in IE11 yet, but I'll find the time by the end of this week and let you know about the results

**Comment by faceplant23** [comment]
Thank you so much in advance!

**Comment by pa7** [comment]
alright, I just checked it on IE11 on a Windows 7 machine and the examples all worked. Anyone else experiencing an issue with IE11 ? @faceplant23 could you give a little more context? are you using any plugins? OS ?

**Comment by faceplant23** [comment]
Dear Patrick,
sorry, it took a while until I got a machine with a working IE 11 configuration. I then tested nearly all IE settings and finally found (at least on my company computers) the relevant entry: **Internet Options > Advanced > Accelerated grphics > Use software rendering instead of GPU rendering**.
In our default installation this one is unchecked, and at least with the current hardware then there is no heatmap visible. When this one is checked (and IE is restarted) the heatmap is visible again.
Maybe it helps others out there with same company-computer-issues..
